rds_schema.sql

```diff
@@ -0,0 +1,57 @@
+DROP TABLE IF EXISTS sources;
+DROP TABLE IF EXISTS reddit_article;
+DROP TABLE IF EXISTS story_keyword_link;
+DROP TABLE IF EXISTS reddit_keyword_link;
+DROP TABLE IF EXISTS keywords;
+DROP TABLE IF EXISTS stories;
+
+CREATE TABLE sources(
+    source_id INT GENERATED ALWAYS AS IDENTITY,
+    source_name TEXT UNIQUE,
+    PRIMARY KEY (source_id)
+);
+
+CREATE TABLE stories(
+    story_id INT GENERATED ALWAYS AS IDENTITY,
+    source_id INT,
+    title TEXT,
+    description TEXT,
+    pub_date TIMESTAMP,
+    media_sentiment FLOAT,
+    PRIMARY KEY (story_id),
+    FOREIGN KEY (source_id) REFERENCES sources(source_id)
+);
+
+CREATE TABLE keywords(
+    keyword_id INT GENERATED ALWAYS AS IDENTITY,
+    keyword TEXT UNIQUE,
+    PRIMARY KEY (keyword_id)
+);
+
+CREATE TABLE reddit_article(
+    re_article_id INT GENERATED ALWAYS AS IDENTITY,
+    re_domain TEXT,
+    re_title TEXT,
+    re_article_url TEXT,
+    re_url TEXT,
```

Comment:
I reckon `re_url` should probably be `UNIQUE`, as each url should only appear once within our database.